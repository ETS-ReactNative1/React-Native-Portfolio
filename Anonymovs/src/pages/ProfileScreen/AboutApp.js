import React from 'react';
import {SafeAreaView, StyleSheet, Text, View, ScrollView} from 'react-native';
import Colors from '../../../assets/config/Colors';
import Icon from 'react-native-vector-icons/MaterialCommunityIcons';

export default function AboutApp() {
  return (
    <ScrollView style={styles.container}>
      <SafeAreaView style={{marginBottom:40}}>
        <View style={styles.desc}>
          <Text style={styles.title}>What is Anonymovs</Text>
          <Text style={styles.description}>
            Anonymovs is a mobile application that i build for Technical Assessment from
            Glints Academy and written and built programmatically in React Native. This app is just a simple collection movies for
            person who want to know all movies that played in the cinema,
            upcaming or top rated. You can also give some rating to a movie and
            add movie to your favorite list.
          </Text>
        </View>
        <View style={styles.desc}>
          <Text style={styles.title}>Features</Text>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Search for a Movies</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Save and Delete a Movie</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Display information of a getAllMovies</Text>
          </View>
        </View>
        <View style={styles.desc}>
          <Text style={styles.title}>Frameworks</Text>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>The MovieDB API</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Vector Icon</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Animatable</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Linear Gradient</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>AirBnB Rating</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Youtube iFrame</Text>
          </View>
        </View>
        <View style={styles.desc}>
          <Text style={styles.title}>Tech</Text>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>Redux Saga</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>XCode & Anroid Studio</Text>
          </View>
          <View style={{flexDirection:'row',alignItems:'center',marginVertical:3}}>
            <Icon style={{marginRight:10}} name="square" size={10} color="white" />
            <Text style={styles.description}>VSCode</Text>
          </View>
        </View>
      </SafeAreaView>
    </ScrollView>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: Colors.background,
    padding: 30,
  },
  desc: {
    marginBottom: 20,
  },
  title: {
    color: Colors.white,
    fontWeight: '700',
    fontSize: 20,
    marginVertical: 10,
  },
  description: {
    color: Colors.white,
  },
  h2: {
    color: Colors.white,
    fontWeight: '500',
    fontSize: 16,
    marginVertical: 10,
  },
});
