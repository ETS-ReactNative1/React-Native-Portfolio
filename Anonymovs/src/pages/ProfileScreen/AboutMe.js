import React from 'react';
import {SafeAreaView, StyleSheet, Text, View} from 'react-native';
import Colors from '../../../assets/config/Colors';

export default function AboutMe() {
  return (
    <View style={styles.container}>
      <SafeAreaView>
        <Text style={styles.h1}>About Me</Text>
        <Text style={styles.h3}>
          A competent IT professional with experience in network and
          troubleshoot software or hardware for 5 years. Extensive experience of
          working in the front line helping clients and colleagues resolve
          complex technical IT issues. Possessing excellent natural problem
          solving and analytical skills and able to contribute to the
          development of best practice, procedures and policy within a company. And just finnishing studied React Native at Glints Academy Batch 11.
        </Text>
      </SafeAreaView>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: Colors.background,
    padding: 30,
  },
  h1: {
    color: Colors.white,
    fontWeight: '700',
    fontSize: 20,
    marginVertical:10
  },
  h3: {
    color: Colors.white,
  },
});
