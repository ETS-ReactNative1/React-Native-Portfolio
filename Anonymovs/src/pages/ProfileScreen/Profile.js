import React from 'react';
import {
  StyleSheet,
  Text,
  View,
  SafeAreaView,
  TouchableOpacity,
} from 'react-native';
import Colors from '../../../assets/config/Colors';
import {useSelector} from 'react-redux';
import {
  widthPercentageToDP as wp,
  heightPercentageToDP as hp,
} from 'react-native-responsive-screen';

export default function Profile({navigation}) {
  const {name} = useSelector(state => state.allReducers);
  return (
    <View style={styles.container}>
      <View style={styles.profile}>
        <Text style={{color: 'white', fontSize: 20}}>Helloo .... {name}</Text>
      </View>
      <View style={styles.contentWrapper}>
        <TouchableOpacity onPress={()=>navigation.navigate('AboutApp')} style={styles.content}>
          <Text style={styles.text}>About this app</Text>
        </TouchableOpacity>
        <TouchableOpacity onPress={()=>navigation.navigate('AboutMe')} style={styles.content}>
          <Text style={styles.text}>About Me</Text>
        </TouchableOpacity>
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: Colors.background,
    justifyContent: 'center',
    alignItems: 'center',
    padding: 30,
  },
  profile: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
  contentWrapper: {
    flex: 1,
  },
  content: {
    backgroundColor: Colors.ash,
    width: wp(80),
    justifyContent: 'center',
    alignItems: 'center',
    padding: wp(3),
    marginVertical: hp(1),
    borderRadius: wp(2),
  },
  text:{
      fontWeight:'500',
      color: Colors.black
  }
});
