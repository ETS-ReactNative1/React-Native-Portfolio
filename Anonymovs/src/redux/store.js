import {createStore, applyMiddleware} from 'redux';
import createSagaMiddleware from 'redux-saga';
import sagas from './saga/sagas';
import combineReducers from './reducer/combine'

const sagaMiddleware = createSagaMiddleware();
const store = createStore(combineReducers,applyMiddleware(sagaMiddleware));

sagaMiddleware.run(sagas);

export default store;