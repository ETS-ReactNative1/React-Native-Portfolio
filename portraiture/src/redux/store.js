import {createStore, applyMiddleware} from 'redux';
import createSagaMiddleware from 'redux-saga';
import combineReducers from './reducers/combiner'
import sagas from './sagas/Sagas'

const sagaMiddleware = createSagaMiddleware();
const store = createStore(combineReducers,applyMiddleware(sagaMiddleware));

sagaMiddleware.run(sagas);

export default store;