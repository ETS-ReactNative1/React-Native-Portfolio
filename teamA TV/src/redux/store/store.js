import ReduxThunk from 'redux-thunk';
import {createStore, applyMiddleware} from 'redux';
import rootReducer from '../reducer/combineReducer'


export let store = createStore(rootReducer,applyMiddleware(ReduxThunk))
