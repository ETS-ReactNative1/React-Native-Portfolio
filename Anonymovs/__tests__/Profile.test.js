import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import store from '../src/redux/store';

import Profile from '../src/pages/ProfileScreen/Profile';
import AboutApp from '../src/pages/ProfileScreen/AboutApp';
import AboutMe from '../src/pages/ProfileScreen/AboutMe';

jest.useFakeTimers();

test('Profile SnapShoot', () => {
  const snap = renderer
    .create(
      <Provider store={store}>
        <Profile />
      </Provider>,
    )
    .toJSON();
  expect(snap).toMatchSnapshot();
});
test('AboutApp SnapShoot', () => {
  const snap = renderer
    .create(
      <Provider store={store}>
        <AboutApp />
      </Provider>,
    )
    .toJSON();
  expect(snap).toMatchSnapshot();
});
test('AboutMe SnapShoot', () => {
  const snap = renderer
    .create(
      <Provider store={store}>
        <AboutMe />
      </Provider>,
    )
    .toJSON();
  expect(snap).toMatchSnapshot();
});
it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Profile />
      </Provider>,
    )
    .toJSON();
  expect(tree.children.length).toBe(2);
});
it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <AboutApp />
      </Provider>,
    )
    .toJSON();
  expect(tree.children.length).toBe(1);
});
it(`renders correctly`, () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <AboutMe />
      </Provider>,
    )
    .toJSON();
  expect(tree.children.length).toBe(1);
});
