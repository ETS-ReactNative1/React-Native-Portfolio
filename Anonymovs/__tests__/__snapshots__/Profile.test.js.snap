// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutApp SnapShoot 1`] = `
<RCTScrollView
  style={
    Object {
      "backgroundColor": "#191926",
      "flex": 1,
      "padding": 30,
    }
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "marginBottom": 40,
        }
      }
    >
      <View
        style={
          Object {
            "marginBottom": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 20,
              "fontWeight": "700",
              "marginVertical": 10,
            }
          }
        >
          What is Anonymovs
        </Text>
        <Text
          style={
            Object {
              "color": "#FFFFFF",
            }
          }
        >
          Anonymovs is a mobile application that i build for Technical Assessment from Glints Academy and written and built programmatically in React Native. This app is just a simple collection movies for person who want to know all movies that played in the cinema, upcaming or top rated. You can also give some rating to a movie and add movie to your favorite list.
        </Text>
      </View>
      <View
        style={
          Object {
            "marginBottom": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 20,
              "fontWeight": "700",
              "marginVertical": 10,
            }
          }
        >
          Features
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Search for a Movies
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Save and Delete a Movie
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Display information of a getAllMovies
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginBottom": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 20,
              "fontWeight": "700",
              "marginVertical": 10,
            }
          }
        >
          Frameworks
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            The MovieDB API
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Vector Icon
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Animatable
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Linear Gradient
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            AirBnB Rating
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Youtube iFrame
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "marginBottom": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 20,
              "fontWeight": "700",
              "marginVertical": 10,
            }
          }
        >
          Tech
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            Redux Saga
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            XCode & Anroid Studio
          </Text>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "marginVertical": 3,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": "white",
                  "fontSize": 10,
                },
                Object {
                  "marginRight": 10,
                },
                Object {
                  "fontFamily": "Material Design Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            󰝤
          </Text>
          <Text
            style={
              Object {
                "color": "#FFFFFF",
              }
            }
          >
            VSCode
          </Text>
        </View>
      </View>
    </RCTSafeAreaView>
  </View>
</RCTScrollView>
`;

exports[`AboutMe SnapShoot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#191926",
      "flex": 1,
      "padding": 30,
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  >
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 20,
          "fontWeight": "700",
          "marginVertical": 10,
        }
      }
    >
      About Me
    </Text>
    <Text
      style={
        Object {
          "color": "#FFFFFF",
        }
      }
    >
      A competent IT professional with experience in network and troubleshoot software or hardware for 5 years. Extensive experience of working in the front line helping clients and colleagues resolve complex technical IT issues. Possessing excellent natural problem solving and analytical skills and able to contribute to the development of best practice, procedures and policy within a company. And just finnishing studied React Native at Glints Academy Batch 11.
    </Text>
  </RCTSafeAreaView>
</View>
`;

exports[`Profile SnapShoot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#191926",
      "flex": 1,
      "justifyContent": "center",
      "padding": 30,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
        }
      }
    >
      Helloo .... 
      
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#999999",
          "borderRadius": 15,
          "justifyContent": "center",
          "marginVertical": 13.5,
          "opacity": 1,
          "padding": 22.5,
          "width": 600,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000000",
            "fontWeight": "500",
          }
        }
      >
        About this app
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#999999",
          "borderRadius": 15,
          "justifyContent": "center",
          "marginVertical": 13.5,
          "opacity": 1,
          "padding": 22.5,
          "width": 600,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000000",
            "fontWeight": "500",
          }
        }
      >
        About Me
      </Text>
    </View>
  </View>
</View>
`;
